website/pages/style.css

```diff
@@ -1,25 +1,44 @@
-@import '@hashicorp/react-global-styles/style.css';
-@import '@hashicorp/react-global-styles/code-highlighting.css';
-@import '@hashicorp/react-global-styles/custom-properties/color.css';
-@import '@hashicorp/react-mega-nav/dist/style.css';
-@import '@hashicorp/react-newsletter-signup-form/dist/style.css';
-@import '@hashicorp/react-text-input/dist/style.css';
-@import '@hashicorp/react-button/dist/style.css';
-@import '@hashicorp/react-footer/dist/style.css';
-@import '@hashicorp/react-consent-manager/dist/style.css';
-@import '@hashicorp/react-toggle/dist/style.css';
-@import '@hashicorp/react-checkbox-input/dist/style.css';
-@import '@hashicorp/react-product-subnav/dist/style.css';
-@import '@hashicorp/react-hero/dist/style.css';
-@import '@hashicorp/react-section-header/dist/style.css';
-@import '@hashicorp/react-use-cases/dist/style.css';
-@import '@hashicorp/react-logo-grid/dist/style.css';
-@import '@hashicorp/react-text-and-content/dist/style.css';
+/* Global Component Styles */
+@import '~@hashicorp/react-global-styles/style.css';
+@import '~@hashicorp/react-global-styles/code-highlighting.css';
+@import '~@hashicorp/react-global-styles/custom-properties/color.css';
+@import '~@hashicorp/react-global-styles/custom-properties/font.css';
+@import '~@hashicorp/react-global-styles/_temporary-to-remove/layout.css';
 
 :root {
   --highlight-color: var(--brand);
 }
 
+@import '~@hashicorp/react-mega-nav/dist/style.css';
+@import '~@hashicorp/react-newsletter-signup-form/dist/style.css';
+@import '~@hashicorp/react-text-input/dist/style.css';
+@import '~@hashicorp/react-button/dist/style.css';
+@import '~@hashicorp/react-footer/dist/style.css';
+@import '~@hashicorp/react-consent-manager/dist/style.css';
+@import '~@hashicorp/react-toggle/dist/style.css';
+@import '~@hashicorp/react-checkbox-input/dist/style.css';
+@import '~@hashicorp/react-product-subnav/dist/style.css';
+@import '~@hashicorp/react-hero/dist/style.css';
+@import '~@hashicorp/react-section-header/dist/style.css';
+@import '~@hashicorp/react-use-cases/dist/style.css';
+@import '~@hashicorp/react-logo-grid/dist/style.css';
+@import '~@hashicorp/react-product-downloader/dist/style.css';
+@import '~@hashicorp/react-vertical-text-block-list/dist/style.css';
+@import '~@hashicorp/react-docs-sidenav/dist/style.css';
+@import '~@hashicorp/react-case-study-slider/dist/style.css';
+@import '~@hashicorp/react-content/dist/style.css';
+@import '~@hashicorp/react-text-and-content/dist/style.css';
+
+/* Local Components */
+@import '../components/docs-page/style.css';
+@import '../components/before-after-diagram/style.css';
+@import '../components/intro-page/style.css';
+
+/* Pages */
+@import './downloads/style.css';
```

Comment:
Can we avoid this step by wrapping all colocated `style.css` files with a `:global { ... }` wrapper

Reply:
I'm not sure I understand, but I'd say likely not since this is not using css modules?

Reply:
Ah, sorry, I was under the impression that css modules were in effect given the `{ experimental: true }` declaration. And I was wondering if all local component and page `style.css` files could be wrapped as `:global` while not being `@imported` here. But on second thought, I guess they'd need to be imported by something.

Ultimately I was thinking it could save a step, but I'm not sure it even would. 👍

Reply:
Ah yes this is not css modules yet - only for the individual pages and components. Files using css modules will be `.module.css` always